doc/active_docs/account_management_api.json

```diff
@@ -9771,15 +9771,15 @@
           },
           "user_key": {
             "type": "string",
-            "description": "User Key (API Key) of the application to be created.",
+            "description": "User Key (API Key) of the application to be created.\n\n**Only alphanumeric characters `[0-9, a-z, A-Z]`, hyphen-minus (`-`), between 5 and 256 characters long. No spaces allowed.**",
             "x-data-threescale-name": "user_keys"
           },
           "application_id": {
             "type": "string",
-            "description": "App ID or Client ID (for OAuth and OpenID Connect authentication modes) of the application to be created."
+            "description": "App ID or Client ID (for OAuth and OpenID Connect authentication modes) of the application to be created.\n\n**Only alphanumeric characters `[0-9, a-z, A-Z]`, hyphen-minus (`-`), between 4 and 256 characters long. No spaces allowed.**"
```

Comment:
Actually, app ID has the same regex as app key `/\A[\x20-\x7E]+\Z/` so it allows more special characters.